vcpkg/commands/upgrade.md

```diff
@@ -0,0 +1,56 @@
+---
+title: vcpkg upgrade
+description: Command line reference for the vcpkg upgrade command.
+ms.date: 4/3/2023
+---
+# vcpkg upgrade
+
+## Synopsis
+
+```console
+vcpkg upgrade [options]
+```
+
+## Description
+
+Remove and reinstalls [Classic mode] packages that have different current available versions.

```

Comment:
I think something should be said here that it's more of a preview of what will be changed, but that it only performs the changes when running with --no-dry-run.